lib/dolly/document.rb

```diff
@@ -98,6 +98,10 @@ def id_as_resource
       CGI::escape id
     end
 
+    def attach_file file_name, mime_type, body
```

Comment:
just one thing as this is a no-reversible action... I will suggest using `attach_file!`  as method name